apps/web/pages/[user].tsx

```diff
@@ -29,40 +31,86 @@ interface EvtsToVerify {
+        </div>
+      </div>
+    </div>
+  ) : (
+    <ul className="space-y-3">
+      {eventTypes.map((type, index) => (
+        <li
+          key={index}
+          className="hover:border-brand group relative rounded-sm border border-neutral-200 bg-white hover:bg-gray-50 dark:border-0 dark:bg-neutral-900 dark:hover:border-neutral-600">
+          <ArrowRightIcon className="absolute right-3 top-3 h-4 w-4 text-black opacity-0 transition-opacity group-hover:opacity-100 dark:text-white" />
+          <Link href={getUsernameSlugLink({ users: props.users, slug: type.slug })}>
+            <a className="flex justify-between px-6 py-4">
+              <div className="flex-shrink">
+                <h2 className="font-cal font-semibold text-neutral-900 dark:text-white">{type.title}</h2>
+                <EventTypeDescription className="text-sm" eventType={type} />
+              </div>
+              <div className="mt-1">
+                <AvatarGroup
+                  border="border-2 border-white"
+                  truncateAfter={4}
+                  className="flex-shrink-0"
+                  size={10}
+                  items={props.users.map((user) => ({
+                    alt: user.name || "",
+                    image: user.avatar || "",
+                  }))}
+                />
+              </div>
+            </a>
+          </Link>
+        </li>
+      ))}
+    </ul>
+  );
   const eventTypeListItemEmbedStyles = useEmbedStyles("eventTypeListItem");
   const query = { ...router.query };
   delete query.user; // So it doesn't display in the Link (and make tests fail)
   useExposePlanGlobally("PRO");
-  const nameOrUsername = user.name || user.username || "";
+  const nameOrUsername = users[0].name || users[0].username || "";
   const [evtsToVerify, setEvtsToVerify] = useState<EvtsToVerify>({});
   return (
     <>
       <Theme />
       <HeadSeo
         title={nameOrUsername}
-        description={(user.bio as string) || ""}
+        description={(users[0].bio as string) || ""}
         name={nameOrUsername}
-        username={(user.username as string) || ""}
+        username={(users[0].username as string) || ""}
         // avatar={user.avatar || undefined}
       />
       <div className="h-screen dark:bg-neutral-900">
         <main className="mx-auto max-w-3xl px-4 py-24">
-          <div className="mb-8 text-center">
-            <AvatarSSR user={user} className="mx-auto mb-4 h-24 w-24" alt={nameOrUsername} />
-            <h1 className="font-cal mb-1 text-3xl text-neutral-900 dark:text-white">
-              <span>{nameOrUsername}</span>
-              {user.verified && (
-                <BadgeCheckIcon className="mx-1 -mt-1 inline h-6 w-6 text-blue-500 dark:text-white" />
-              )}
-            </h1>
-            <p className="text-neutral-500 dark:text-white">{user.bio}</p>
-          </div>
+          {props.users.length < 2 && (
```

Comment:
Move 2 to a constant so we get its name use

Reply:
Yeah, I think I should rather do it `props.users.length === 1` here instead. This is only meant for legacy use and so it needs to be called when it isn't a dynamic group booking. I'll updated it so that it makes a bit more sense.

Reply:
I don't think this merits to be moved to a constant, just that my check was too vague. It should make more sense now. Thanks for pointing this out :)